rasa/core/actions/action.py

```diff
@@ -115,11 +119,21 @@ def combine_with_templates(
     return actions + unique_template_names
 
 
+def is_retrieval_action(action_name: Text, retrieval_intents: List[Text]) -> bool:
+
+    return (
+        True
+        if retrieval_intents and action_name.split(UTTER_PREFIX)[1] in retrieval_intents
+        else False
+    )
```

Comment:
```suggestion
    return retrieval_intents and action_name.split(UTTER_PREFIX)[1] in retrieval_intents
```

Reply:
if `retrieval_intents` can't be `None`, I'd even remove the `retrieval_intents and` part (as the second part of the condition will still be false)